zone/entity.cpp

```diff
@@ -5647,6 +5647,8 @@ void EntityList::StopMobAI()
 
 void EntityList::SendAlternateAdvancementStats() {
 	for (auto &c : client_list) {
+		c.second->Message(0, "Reloading AA");
```

Comment:
`Chat::White`.

Reply:
Good catch, that was a Copilot suggestion :)